src/call/github-events-to-s3.ts

```diff
@@ -12,30 +12,31 @@
 
 import { Probot } from 'probot';
 import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
-import { Resource } from '../service/resource/resource';
-import { validateResourceConfig } from '../utility/verification/verify-resource';
 
-export default async function githubEventsToS3(app: Probot, context: any, resource: Resource): Promise<void> {
-  if (!(await validateResourceConfig(app, context, resource))) return;
+export default async function githubEventsToS3(app: Probot, context: any): Promise<void> {
+  // Removed validateResourceConfig to let this function listen on all repos, and filter for only the repos that are public.
+  // This is done so when a new repo is made public, this app can automatically start processing its events.
+  // if (!(await validateResourceConfig(app, context, resource))) return;
+  if (context.payload.repository?.private === false) {
```

Comment:
Wonder if we need an else branch to log which repo is skipped due to being private.